Sources/LocationMessageCell.swift

```diff
@@ -69,11 +70,32 @@ open class LocationMessageCell: MessageCollectionViewCell<UIImageView> {
 
         let snapShotter = MKMapSnapshotter(options: snapshotOptions)
         snapShotter.start { (snapshot, error) in
-            self.activityIndicator.stopAnimating()
-            if error == nil {
-                self.messageContentView.image = snapshot?.image
+
+            guard let snapshot = snapshot, error == nil else {
+                //show an error image?
+                return
```

Comment:
Need to call `stopAnimating()` here

Reply:
We could use defer {} to never forget again 😬